price-feeder/config/config.go

```diff
@@ -158,15 +158,26 @@ func ParseConfig(configPath string) (Config, error) {
 		cfg.Server.ReadTimeout = defaultSrvReadTimeout.String()
 	}
 
+	pairs := make(map[string]map[string]struct{})
 	for _, cp := range cfg.CurrencyPairs {
 		if !strings.Contains(strings.ToUpper(cp.Quote), denomUSD) {
 			return cfg, fmt.Errorf("unsupported pair quote: %s", cp.Quote)
 		}
+		if _, ok := pairs[cp.Base]; !ok {
+			pairs[cp.Base] = make(map[string]struct{})
+		}
 
 		for _, provider := range cp.Providers {
 			if _, ok := SupportedProviders[provider]; !ok {
 				return cfg, fmt.Errorf("unsupported provider: %s", provider)
 			}
+			pairs[cp.Base][provider] = struct{}{}
+		}
+	}
+
+	for base, providers := range pairs {
+		if _, ok := pairs[base]["mock"]; !ok && len(providers) < 3 {
```

Comment:
This just evaluates to ignoring assets that have a mock provider enabled?

Reply:
Yup, since e2e uses mock